shadow.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: "com.github.johnrengelman.shadow"
+
+configurations {
+    shade
```

Comment:
why not `shadow`? this isn't maven.

Reply:
shadow defines the configuration `shadow` and freaks out if you try to steal it from it

Reply:
it defines the configuration -- you should be able to just go ahead and reuse that shadow configuration when configuring the shadow tasks.